src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -4,6 +4,7 @@
     <RootNamespace>System.Numerics</RootNamespace>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
@eerhardt, could you confirm if this is the desired way to do this?

Reply:
Typically we *try* to avoid `#if` and `DefineConstants` in our code, and instead opt for whole files that are included for each target framework.  However, in this case, I don't see a good way to do that unfortunately.  (If you can find a good way, feel free to use that approach.)

If we do need to use `DefineConstants`, it is preferred to use constants that are tied to the concept that you need, and not which target happens to have that capability today.  For example, if you need to use the Registry, you'd have a constant for `HAS_REGISTRY` or `FEATURE_REGISTRY`, instead of having `#if WINDOWS`.

So in this case `HAS_INTRINSICS` seems more preferred, than `NETCOREAPP`.  See https://github.com/dotnet/corefx/commit/c3c2faac6f691a3f1520e616794da0af630b7d51#diff-a50daad0581f9c1c7d0d39fc82cd9457L304 for an example where we are using Span.

Reply:
@eerhardt  Got it! Are intrinsics available for CoreRT?

Reply:
Not today.